src/software/simulation/threaded_simulator_test.cpp

```diff
@@ -198,7 +198,7 @@ TEST_F(ThreadedSimulatorTest, add_robots_and_primitives_while_simulation_running
     threaded_simulator.setYellowRobotPrimitives(yellow_primitives_ptr);
 
     std::this_thread::yield();
-    std::this_thread::sleep_for(std::chrono::milliseconds(2000));
+    std::this_thread::sleep_for(std::chrono::milliseconds(2500));
```

Comment:
What's the reason for increasing this?

Reply:
The memory leak check is done in -c dbg mode so that errors more human readable and that slows it down enough that the other thread in this `ThreadedSimulatorTest` doesn't have enough time to get the robots in the correct position